pyiceberg/catalog/rest.py

```diff
@@ -773,6 +771,17 @@ def _commit_table(self, table_request: CommitTableRequest) -> CommitTableRespons
             )
         return CommitTableResponse(**response.json())
 
+    @retry(**_RETRY_ARGS)
+    def list_views(self, namespace: Union[str, Identifier]) -> List[Identifier]:
```

Comment:
nit: move `list_views` above `commit_table` for better github diff

Reply:
Thanks, and done! This one caused a merge conflicts when pulling in main while working on this PR 👍